packages/fx-core/src/core/question.ts

```diff
@@ -87,9 +89,10 @@ export enum CoreQuestionNames {
   ReplaceWebsiteUrl = "replaceWebsiteUrl",
   AppPackagePath = "appPackagePath",
   ReplaceBotIds = "replaceBotIds",
-  TeamsAppManifestFilePath = "teamsAppManifestFilePath",
+  TeamsAppManifestFilePath = "manifest-path",
```

Comment:
CLI parameter should follow the same pattern.

Reply:
The existed cli command update Teams/aad app uses "manifest-file-path" as param name
![image](https://user-images.githubusercontent.com/71362691/226792534-0041e484-2fd2-4d4d-a0f6-4e3c526c8ef7.png)